cachi2/core/package_managers/yarn/main.py

```diff
@@ -5,6 +5,7 @@
 from cachi2.core.models.output import Component, EnvironmentVariable, RequestOutput
 from cachi2.core.package_managers.yarn.project import (
     Project,
+    YarnRc,
```

Comment:
Ouch. Disallowing zero-installs will make our support of yarn even more limited, since we're already banning plugins, git deps, exec locators...

I think it's definitely worth discussing, but I'd say let's stick to the plan (and keep these changes here) until we figure out a new one :)

Reply:
Alright, let's discuss it in a separate RFC then and keep this as is.